backend/backend/EntityConfigurations/UserEntityConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿using backend.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System.Collections.Generic;
+
+namespace backend.EntityConfigurations
+{
+    public class UserEntityConfiguration : IEntityTypeConfiguration<User>
+    {
+        private List<User> Users;
+        public UserEntityConfiguration(List<User> users)
+        {
+            Users = users;
+        }
+        public void Configure(EntityTypeBuilder<User> builder)
+        {
+            builder.HasKey(c => c.Id);
+            builder.Property(c => c.Firstname).HasMaxLength(20).IsRequired();
+            builder.Property(c => c.Lastname).HasMaxLength(20).IsRequired();
+            builder.Property(c => c.Email).HasMaxLength(30).IsRequired();
```

Comment:
```suggestion
            builder.Property(c => c.Email).HasMaxLength(256).IsRequired();
```
Nereiktų ribot email ilgio, apart sanity ribų (256 pvz)